UDV-Core/src/Utils/3DTiles/3DTilesUtils.js

```diff
@@ -167,6 +167,191 @@ export function setTileVerticesColor(tile, newColor, indexArray = null) {
   }
 }
 
+/**
+ * Creates tile groups and associate them with the given materials.
+ * 
+ * @param {*} tile The 3DTiles tile.
+ * @param {Array<any>} materials An array of material parameters. Each entry
+ * should be an object that will be passed as parameter in the THREE.Material
+ * constructor. Accepted values are for example `color` or `opacity` (see
+ * THREE.js documentation).
+ * @param {Array<any>} ranges An array of ranges. A range is an object with 3
+ * propeties :
+ * - `start`: the start index of the group of vertices
+ * - `count`: the number of vertices of the group
+ * - `material`: the index of the material in the materials array
+ */
```

Comment:
Could you add an @example similar to the one of createTileGroupsFromBatchIDs() please?

Reply:
Done